src/pages/Home/Home.scss

```diff
@@ -0,0 +1,236 @@
+@use '../../styles/fonts' as *;
+@use '../../styles/variables' as *;
+@use '../../styles/mixins' as *;
+
+.home {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  position: relative;
+  background-color: $color-primary;
+}
+
+.user-prompts{
+  background-color: $color-offset-secondary;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  
+  @include tablet {
+    flex-direction: row;
+    max-height: 65vw;
+  }
+
+  @include desktop {
+    max-height: 90vh;
+  }
+}
+
+.svg__top-curve{
+  width: 100%;
+  height: 100%;
+  position: relative;
+  top: .2%;
+  // Top necessary to remove 'line between svg and next div'
+
+  @include tablet{
+    display: none;
+  }
+
+  & path {
+    min-width:320px;
+  }
+}
+
+.svg__top-curve--clip-path{
+  width: 100%;
+  z-index: -1;
+  position: absolute;
+  display: none;
+
+  @include tablet{
+    display: flex;
+  }
+}
+
+// Uncertain if we'll end up using this search bar elsewhere in the app, if we do
```

Comment:
Brainstorm best way to log these? 
I'm concerned comments in code are easily lost in the future.

Reply:
To be reviewed at a later date. For this specific case, as it was already being used in two locations  it has been moved to global styles next to the button.